### PR TITLE
RDOIAT-710: Upgrade cefglue version

### PR DESCRIPTION
**Pull request by cangosta**
…sues; Adapt to changes in cef glue API

### CODE REVIEW

File: Tests.WebView/IsolatedJavascriptEvaluation.cs [hunk 1]
```diff
@@ -186,5 +186,13 @@ public class IsolatedJavascriptEvaluation : WebViewTestBase {
                 Assert.AreEqual(2, y);
             });
         }
+
+        private string EnsureObjectIsBoundWrapper(string objName, string testCode) {
```
Comment (cangosta) [inline Tests.WebView/IsolatedJavascriptEvaluation.cs:190]: Temporary just for POC

File: Tests.WebView/IsolatedJavascriptEvaluation.cs [hunk 2]
```diff
@@ -49,8 +50,12 @@ public class IsolatedJavascriptEvaluation : WebViewTestBase {
                     interceptorCalled = true;
                     return originalFunc();
                 }
-                TargetView.RegisterJavascriptObject(DotNetObject, functionToCall, Interceptor);
-                await Load($"<html><script>{DotNetObject}.invoke();</script><body></body></html>");
+                TargetView.RegisterJavascriptObject(DotNetObject, functionToCall, Interceptor);
+
+                var script = $"{EnsureObjectsAreBound(DotNetObject)}; {DotNetObject}.invoke();";
+                var html = CreateHtmlDocumentWithScript(script);
+                await Load(html);
```
[the remark this answers is not in the document]
Comment (cangosta) [reply Tests.WebView/IsolatedJavascriptEvaluation.cs]: done

File: SampleWebView.Avalonia/SampleWebView.Avalonia.csproj [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>WinExe</OutputType>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```
[the remark this answers is not in the document]
Comment (cangosta) [reply SampleWebView.Avalonia/SampleWebView.Avalonia.csproj]: done

File: Tests.WebView/WebViewTestBase.cs [hunk 4]
```diff
@@ -1,11 +1,14 @@
...
+using System.Linq;
 using System.Threading.Tasks;
 using NUnit.Framework;
 using WebViewControl;
 
 namespace Tests.WebView {
 
-    public class WebViewTestBase : TestBase<WebViewControl.WebView> {
+    public class WebViewTestBase : TestBase<WebViewControl.WebView> {
+        private List<string> registeredNativeObjects = new();
```
[the remark this answers is not in the document]
Comment (cangosta) [reply Tests.WebView/WebViewTestBase.cs]: done

File: Tests.WebView/WebViewTestBase.cs [hunk 5]
```diff
@@ -63,6 +73,19 @@ public class WebViewTestBase : TestBase<WebViewControl.WebView> {
             }
             TargetView.ShowDeveloperTools();
             return true;
+        }
+
+        protected void RegisterJavascriptObject(string name, object objectToBind,
+            Func<Func<object>, object> interceptCall = null) {
```
[the remark this answers is not in the document]
Comment (cangosta) [reply Tests.WebView/WebViewTestBase.cs]: done

**Review by joaompneves — APPROVED** [review]

**Review by joaompneves — APPROVED** [review]

File: WebViewControl.Avalonia/WebViewControl.Avalonia.csproj [hunk 6]
```diff
@@ -1,23 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>$(DotnetStandardVersion)</TargetFramework>
```
Comment (joaompneves) [inline WebViewControl.Avalonia/WebViewControl.Avalonia.csproj:4]: put everything on .net6

Comment (cangosta) [reply WebViewControl.Avalonia/WebViewControl.Avalonia.csproj]: done

**Review by joaompneves — APPROVED** [review]

**Review by osesantos — APPROVED** [review]

**Pull request merged**